docs/configuration/metrics/redis-cache.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Azure Cache for Redis Declaration
+---
+
+## Azure Cache for Redis - ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v0.1.0-green.svg)
```

Comment:
It's actually only available starting v1.0

```suggestion
## Azure Cache for Redis - ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.0.0-green.svg)
```

Reply:
done.